projects/buck2/build.mk

```diff
@@ -0,0 +1,50 @@
+# --------------------------------------------------------------------------------
+# Project: Buck2 Build Script
+# Description: This script is used for building and managing the Buck2 project.
+# Author: Christian Abrokwa
+# --------------------------------------------------------------------------------
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+
+$(eval $(call project-define,buck2))
+
+PROJECT_NAME := buck2
+PROJECT_VERSION := 2023-09-01
+
+HOST_OUT_DIR := out/host/bin
```

Comment:
We already have a variable named `HOST_OUT_DIR`, there also is `HOST_BUILD_DIR` which is a very similar name to `BUILD_HOST_DIR`. Maybe we can avoid defining those two ?

Reply:
Sure.